src/wazuh_qa_framework/system/host_manager.py

```diff
@@ -632,6 +632,22 @@ def install_package(self, host, package_name, become=None, ignore_errors=False):
 
         return result
 
+    def stop_ungracefully_process(self, host, process):
```

Comment:
Some suggestions for the name.

```suggestion
    def process_basic_stop(self, host, process):
```

```suggestion
    def process_simple_stop(self, host, process):
```

Reply:
The `ungracefully` term was taken from issues like https://github.com/wazuh/wazuh/issues/15153#issue-1404640786 where the host connection is crashed or lost. IMHO I wish to keep it because it is a simple word that reflects that situation.